base/process.jl

```diff
@@ -381,7 +381,7 @@ const SpawnIOs = Vector{Any} # convenience name for readability
             h === C_NULL     ? (0x00, UInt(0)) :
             h isa OS_HANDLE  ? (0x02, UInt(cconvert(@static(Sys.iswindows() ? Ptr{Cvoid} : Cint), h))) :
             h isa Ptr{Cvoid} ? (0x04, UInt(h)) :
-            error("invalid spawn handle $h from $io")
+            error("invalid spawn handle $h from $io") #TODO use a real error type here
```

Comment:
`error` is perfectly acceptable for errors, let's not add a TODO suggesting otherwise (this one is nearly an `@assert` on `rawhandle` giving the expected result)

Reply:
Earlier I had changed it to an `@assert`, somewhere in the history can find me changing it back.

Reply:
Done